_topic_maps/_topic_map_osd.yml

```diff
@@ -251,6 +251,83 @@ Topics:
     - Name: Configuring custom domains for applications
       File: osd-config-custom-domains-applications
 ---
+Name: Logging
+Dir: logging
+Distros: openshift-dedicated
+Topics:
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
+  File: cluster-logging
+- Name: Installing AWS CloudWatch logging add-on service
+  File: sd-install-cloudwatch-logging
+- Name: Accessing the service logs
```

Comment:
I think this one also needs conditionals. It's building as ROSA in the OSD docs:

Build preview - https://53099--docspreview.netlify.app/openshift-dedicated/latest/logging/rosa-accessing-the-service-logs.html

Current OSD file - https://docs.openshift.com/dedicated/osd_cluster_admin/osd_logging/osd-accessing-the-service-logs.html